### PR TITLE
Update poll icons from Compound

### PR DESCRIPTION
**Pull request by julioromano**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [x] Technical
- [ ] Other :

## Content

Update poll icons from Compound, remove the old ones.

## Motivation and context

Story: https://github.com/vector-im/element-meta/issues/2182

## Tests

- Step 1
- Step 2
- Step ...

## Tested devices

- [ ] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-x-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR

### CODE REVIEW

**Comment by julioromano** [comment]
Screenshots job: https://github.com/vector-im/element-x-android/actions/runs/6954501748

**Review by jmartinesp — APPROVED** [review]

File: tests/uitests/src/test/snapshots/images/ui_S_t[f.poll.api_PollContentCreatorEnded_null_PollContentCreatorEnded-Day-12_13_null,NEXUS_5,1.0,en].png
Comment (bmarty) [inline tests/uitests/src/test/snapshots/images/ui_S_t[f.poll.api_PollContentCreatorEnded_null_PollContentCreatorEnded-Day-12_13_null,NEXUS_5,1.0,en].png:1]: Hard to see the diff here 🤔

**Review by bmarty — APPROVED** [review]
Thanks!

File: tests/uitests/src/test/snapshots/images/ui_S_t[f.poll.api_PollContentCreatorEnded_null_PollContentCreatorEnded-Day-12_13_null,NEXUS_5,1.0,en].png
Comment (julioromano) [reply tests/uitests/src/test/snapshots/images/ui_S_t[f.poll.api_PollContentCreatorEnded_null_PollContentCreatorEnded-Day-12_13_null,NEXUS_5,1.0,en].png:1]: The rounding of the bars seem a little bit different....

@amshakal can you just give an eye on this screenshots to see if the new poll icons look okay?

**Comment by julioromano** [comment]
Updated screenshot incoming after rebase: https://github.com/vector-im/element-x-android/actions/runs/6955630430

**Pull request merged**